### PR TITLE
About section

### PR DESCRIPTION
**Pull request by yusufroqib**
@DevBigEazi. I have implemented the about section design. #16

### CODE REVIEW

File: src/sections/About.jsx [hunk 1]
```diff
@@ -1,5 +1,26 @@
+import { icons, images } from "../constants";
+
 const About = () => {
-  return <section>About</section>;
+	return (
+		<section className="bg-[#0F0140] max-container grid py-12 md:grid-cols-2 gap-4 text-[#F2EDE4]">
```
Comment (DevBigEazi) [inline src/sections/About.jsx:5]: Laverage the configurations(colors) in tailwind.config.js

**Review by DevBigEazi — COMMENTED** [review]
@yusufroqib

**Review by DevBigEazi — CHANGES_REQUESTED** [review]
quickly implement this pls

File: src/sections/About.jsx (hunk 1 above)
Comment (yusufroqib) [reply src/sections/About.jsx]: @DevBigEazi , I have fixed it.

**Pull request merged**